pkgs/development/python-modules/colbert-ai/default.nix

```diff
@@ -0,0 +1,66 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  bitarray,
+  datasets,
+  flask,
+  python-dotenv,
+  ninja,
+  scipy,
+  tqdm,
+  transformers,
+  ujson,
+  gitpython,
+  torch,
+  faiss,
+}:
+
+buildPythonPackage rec {
+  pname = "colbert-ai";
+  version = "0.2.21";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "colbert_ai";
+    hash = "sha256-qNb9tOInLysI7Tf45QlgchYNhBXR5AWFdRiYt35iW6s=";
+  };
+
+  pythonRemoveDeps = [ "git-python" ];
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    bitarray
+    datasets
+    flask
+    gitpython
+    python-dotenv
+    ninja
+    scipy
+    tqdm
+    transformers
+    ujson
+    torch
+    faiss
+  ];
+
+  pythonImportsCheck = [ "colbert" ];
+
```

Comment:
Could you enable tests?

> Make sure the tests are enabled using for example [pytestCheckHook](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook) and, in the case of libraries, are passing for all interpreters. If certain tests fail they can be disabled individually. Try to avoid disabling the tests altogether. In any case, when you disable tests, leave a comment explaining why.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
There's no tests :(

Reply:
> In any case, when you disable tests, leave a comment explaining why.

Please leave a comment.